test/norm_flows.jl

```diff
@@ -0,0 +1,22 @@
+using Test
+using Bijectors, ForwardDiff, LinearAlgebra
+
+@testset "planar flows" begin
+    for i in 1:100
+        flow = PlanarLayer(10)
+        z = randn(10,100)
+        forward_diff = log(abs(det(ForwardDiff.jacobian(t -> transform(flow, t).data, z))))
+        our_method = sum(forward(flow, z).logabsdetjacob)
+        @test our_method ≈ forward_diff
+    end
+end
+
+@testset "radial flows" begin
```

Comment:
As you've noted, this test is failing. But I don't it's due to numerical errors. If you run the below code you'll the values in the arrays are _very_ different.
```
flow = RadialLayer(10)
z = randn(10,100)
forward_diff = log.(abs.(det.([ForwardDiff.jacobian(t -> transform(flow, t).data, z[:, j]) for j = 1:100])))
our_method = vec(Tracker.data.(forward(flow, z).logabsdetjacob.data))
```

Reply:
If this is the case the Radial forward is likely to be buggy.

Reply:
@sharanry Sorry, but is this fixed? Have you checked my comment above and verified that this now works?

Reply:
Yes, Radial flows is fixed now. The problem was with a missed step in the `inv()` function.

Reply:
Hmm, but I'm not using `inv` in the code above, right?
I got wildly different results for those, have you verified the exact code above?

Reply:
I think Sharan didn't completely sync his notebook code which we worked on together and this branch. At least the axis `norm` should act on seems to be wrong for me (one of my new comments).

Reply:
@sharanry Can you make sure all tests pass on your local?